cmd/blockchaincmd/deploy.go

```diff
@@ -124,7 +124,7 @@ so you can take your locally tested Blockchain and deploy it on Fuji or Mainnet.
 	cmd.Flags().StringVar(
 		&userProvidedAvagoVersion,
 		"avalanchego-version",
-		constants.DefaultAvalancheGoVersion,
+		constants.FujiAvalancheGoV113,
```

Comment:
changing FujiAvalancheGoV113 to new version is better, since we already have this 
https://github.com/ava-labs/avalanche-cli/blob/3fee311fe93caa2a2e2a7ffd17b8d8f44c3a9603/cmd/blockchaincmd/deploy.go#L488